source/common/json/config_schemas.cc

```diff
@@ -1403,6 +1403,11 @@ const std::string Json::Schema::CLUSTER_HEALTH_CHECK_SCHEMA(R"EOF(
         "minimum" : 0,
         "exclusiveMinimum" : true
       },
+      "interval_jitter_percent" : {
```

Comment:
v1 config support is deprecated, and we're not adding new features to it; please revert this file (and the related test).